msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/ClientAssertion.java

```diff
@@ -13,7 +12,7 @@
 @EqualsAndHashCode
 final class ClientAssertion implements IClientAssertion {
 
-    static final String assertionType = JWTAuthentication.CLIENT_ASSERTION_TYPE;
+    static final String ASSERTION_TYPE = "urn:ietf:params:oauth:client-assertion-type:jwt-bearer";
```

Comment:
Rename to `ASSERTION_TYPE_JWT_BEARER` ? There will be others soon.

Reply:
Renamed in the latest commit.